ddtrace/ext/ci.py

```diff
@@ -292,6 +326,17 @@ def extract_github_actions(env):
     if run_attempt:
         pipeline_url = "{0}/attempts/{1}".format(pipeline_url, run_attempt)
 
+    # OrderedDict is necessary for comparing against a fixture in testing
+    env_vars = OrderedDict(
+        [
+            ("GITHUB_SERVER_URL", env.get("GITHUB_SERVER_URL")),
+            ("GITHUB_REPOSITORY", env.get("GITHUB_REPOSITORY")),
+            ("GITHUB_RUN_ID", env.get("GITHUB_RUN_ID")),
+        ]
+    )
+    if env.get("GITHUB_RUN_ATTEMPT") is not None:
+        env_vars["GITHUB_RUN_ATTEMPT"] = env.get("GITHUB_RUN_ATTEMPT")
```

Comment:
```suggestion
        env_vars["GITHUB_RUN_ATTEMPT"] = env["GITHUB_RUN_ATTEMPT"]
```

We don't have to use `.get()` since we know it is there already. (`[]` access is also a little bit faster)

Reply:
(nit pick, not required)